collectors/python.d.plugin/bigquery/README.md

```diff
@@ -0,0 +1,62 @@
+<!--
+title: "BigQuery"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/python.d.plugin/bigquery/README.md
+-->
+
+# BigQuery Netdata Collector
+
+A python based collector leveraging [`pandas-gbq`](https://pandas-gbq.readthedocs.io/en/latest/) to run a query 
+against [BigQuery](https://cloud.google.com/bigquery) and save results into a chart in Netdata.
+
+## Requirements
+
+This collector depends on some Python packages that can usually be installed via `pip`.
```

Comment:
```suggestion
This collector depends on some Python (Python 3 only) packages that can usually be installed via `pip` or `pip3`.
```

Reply:
Perfect, with this I think the PR will be ready to be merged. :) 
What do you think @Ferroin ?